### PR TITLE
fit grammar consistency

### LINKED ISSUE #12694: Grammar Inconsistency
**Describe the bug**
When saving permissions, the "Save Changes" button has a capital C. Everywhere else in the site it's lowercase.

**To Reproduce**
1. Go to '/admin/permissions/databases'
2. Click on data access for one of your groups (either revoking or granting access)
3. In the upper right-hand corner, the save button says "Save Changes, but it should say "Save changes" for website consistency.

**Expected behavior**
Expected the C to be lowercase 😅

**Screenshot**
Screen Shot 2020-06-12 at 3 13 20 PM

**Severity**
So mild I'm not sure it's worth mentioning, but I'm a word nerd

### PR DESCRIPTION
**Pull request by linzichao**
fix #12694

###### Before submitting the PR, please make sure you do the following
- [ ] If you're attempting to fix a translation issue, please submit your changes to our [POEditor project](https://poeditor.com/join/project/ynjQmwSsGh) instead of opening a PR.
### Tests
-  [x] Run the frontend and integration tests with  `yarn lint && yarn flow && yarn test`)
-  [x] If there are changes to the backend codebase, run the backend tests with `lein test && lein lint && ./bin/reflection-linter`

-  [x] Sign the [Contributor License Agreement](https://docs.google.com/a/metabase.com/forms/d/1oV38o7b9ONFSwuzwmERRMi9SYrhYeOrkbmNaq9pOJ_E/viewform)
(unless it's a tiny documentation change).

### CODE REVIEW

File: frontend/src/metabase/css/components/header.css [hunk 1]
```diff
@@ -67,7 +67,6 @@
   color: white;
   border: none;
   font-size: 1em;
-  text-transform: capitalize;
```
Comment (salsakran) [inline frontend/src/metabase/css/components/header.css:70]: @kdoh does removing this make sense?

Comment (kdoh) [reply frontend/src/metabase/css/components/header.css:70]: @salsakran Yup. It makes sense in the context of the other change. Basically going from `One Two` to `One two` and making sure CSS doesn't overwrite that.

**Review by salsakran — APPROVED** [review]

**Review by kdoh — APPROVED** [review]

**Review by salsakran — CHANGES_REQUESTED** [review]
Spoke too soon. 

@linzichao  -
This breaks our Cypress tests. Please update the tests to reflect this change in the string and run them locally prior to resubmitting this PR. It's more work to fix the tests than to change one character so this PR isn't that helpful if we need to clean up after it.

**Comment by linzichao** [comment]
@salsakran 
I made a slight change on Cypress tests. Hope it might be helpful.

File: frontend/src/metabase/css/components/header.css (hunk 1 above)
Comment (nemanjaglumac) [reply frontend/src/metabase/css/components/header.css:70]: How many other strings does this affect?

**Comment by salsakran** [comment]
@linzichao you did not re-run the test suite. If you had, you would have found other tests that broke. 

Cypress tests do not run for 3rd party PRs (as we use private resources like proprietary database servers whose credentials we do not want to leak). We expect you to run the full test suite locally when you work on PRs.

Please do not submit PRs without running the full test suite and making sure that it continues to work. This is a simple change (one character) that requires us to fix many tests to get our test suite passing again. It's an absolutely valid PR, and we encourage you to find small issues like this and help out. 

However, it is an absolutely hard requirement that any PR not break out tests.

**Pull request closed without merging**